agent/functional_tests/util/utils_unix.go

```diff
@@ -166,8 +166,12 @@ func (agent *TestAgent) StartAgent() error {
 		Links: agent.Options.ContainerLinks,
 	}
 
+	if os.Getenv("ECS_FTEST_FORCE_NET_HOST") != "" {
```

Comment:
Where is this environment variable being set?

Reply:
It's not automatically set.  I added this because I was developing the test in an environment where the docker0 bridge did not have Internet access (for unrelated reasons) and wanted the agent to actually be able to run.

Reply:
Then can you add this into the `agent/functional_tests/README.d`.

Reply:
Done.  I rewrote a bit more of the README to make it more clear.